topi/include/topi/elemwise.h

```diff
@@ -169,5 +171,110 @@ inline Tensor cast(const Tensor& x,
   }, name, tag);
 }
 
+/*!
+* \brief Creates an operation that sum each element of a tensor
+*
+* \param x The input tensor
+* \param num_args The number of arguments
+*
+* \return A Tensor whose op member is the sum operation
+*/
+inline Tensor elemwise_sum(const Array<Tensor>& xs,
+                           int num_args,
+                           std::string name = "tensor",
+                           std::string tag = kElementWise) {
+  CHECK_GT(xs.size(), 0) << "elemwise sum must have at least one input tensor.";
+  return compute(xs[0]->shape, [&](const Array<Var>& i) {
+      auto sum_expr = xs[0](i);
+      for (int j = 1; j < xs.size(); j++) {
+        sum_expr = sum_expr + xs[j](i);
+      }
+      return sum_expr;
+  }, name, tag);
+}
+
+/*!
+* \brief Creates an operation that fill a tensor with fill_value
+*
+* \param shape The shape of a tensor
+* \param dtype The number of arguments
+* \param fill_value The value to be filled
+*
+* \return A Tensor filled with fill_value
+*/
+inline Tensor full(const Array<Expr>& shape,
+                   Type dtype,
+                   Expr fill_value,
+                   std::string name = "tensor",
+                   std::string tag = kElementWise) {
+  Expr ev = lossless_cast(dtype, fill_value);
```

Comment:
It's possible that lossless_cast could return an undefined Expr.

Could raise an appropriate error here to user !